data/json/professions.json

```
@@ -3651,14 +3651,14 @@
     "type": "profession",
     "id": "national_guard",
     "name": "National Guard",
-    "description": "Your National Guard unit was activated when the epidemic struck.  Despite your best efforts you did not manage to meet up with them before all communications ceased and you found yourself alone amongst the dead.",
+    "description": "Your National Guard unit was activated when the epidemic struck.  Despite your best efforts you did not manage to meet up with them before all communications ceased and you found yourself alone amongst the dead.  Now you're left wishing you got to keep the gun alongside the uniform.",
     "points": 3,
```

```suggestion
    "points": 2,
```
National guard's points worth was already dubious enough, but I feel this especially warrants knocking the points down (unless some other solution is taken to make it better, preferably making it more worth the points than it was even before this PR)

---

I'm gonna go ahead and commit this until a better idea is submitted